easybuild/easyconfigs/f/FlexiBLAS/FlexiBLAS-3.0.4-GCC-10.3.0.eb

```diff
@@ -11,6 +11,13 @@ toolchain = {'name': 'GCC', 'version': '10.3.0'}
 local_extra_flags = "-fstack-protector-strong -fstack-clash-protection"
 toolchainopts = {'pic': True, 'extra_cflags': local_extra_flags, 'extra_fflags': local_extra_flags}
 
+if ARCH == 'ppc64le':
```

Comment:
I don't want to stall this, but in general I think it is better if we avoid logic in easyconfigs if possible (since they are not supposed to contain any). Instead you could create a `local_configopts` dict and just add `local_configopts[ARCH]` to the component.
I say this because this easyconfig will now fall over with some of the `try-*` options.

Reply:
BTW ARCH is not defined is it? Shouldn't it be `%(arch)s`?